docs/react/README.md

```diff
@@ -1,72 +1,186 @@
 # @hatchifyjs/react
 
-- [What is @hatchifyjs/react](#what-is-hatchifyjsreact)
-  - [TypeScript](#typescript)
-  - [Types](#types)
+@hatchifyjs/react is a schema-driven library of components and helper methods for a Hatchify + React frontend. By providing the schemas (AKA models) of your backend resources, @hatchifyjs/react returns a set of components and helper functionality that you can use across your React app.
+
+`@hatchifyjs/react` is a TypeScript supported [NPM package](https://www.npmjs.com/package/@hatchifyjs/react) that takes your [JSON:API rest client](#createjsonapiclient) and[Schema(s)](../schema/README.md) to produce custom:
+
+- components
+- helper functions
+- generated [types](#types)
+
+The following uses `hatchifyReact`to create a `hatchedReact` app with the defined `Todo` and `User` schemas.
+
+```tsx
+import { hatchifyReact, HatchifyProvider, createJsonapiClient } from "@hatchifyjs/react"
+
+// Define your Schema(s)
+export const Schemas = {
+  Todo: {
+    name: "Todo",
+    attributes: {
+      name: string(),
+      status: enumerate({ values: ["Pending", "Failed", "Completed"] }),
+    },
+    relationships: {
+      user: belongsTo("User"),
+    },
+  },
+  User: {
+    name: "User",
+    attributes: {
+      name: string(),
+    },
+    relationships: {
+      todos: hasMany(),
+    },
+  },
+} satisfies Record<string, PartialSchema>
+
+// Create your Hatched React App instance
+const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
+
+// Define variables for your Hatchify schema generated components
+const TodoDataGrid = hatchedReact.components.Todo.DataGrid
```

Comment:
// Define variables for your Hatchify schema generated components

can just be

// Grab the Todo DataGrid component from the hatchedReact object